input/pagecontent/authentication.md

```diff
@@ -0,0 +1,169 @@
+- Adhere to privacy-by-design and security-by-design principles.
+- Leverage existing identity solutions such as x.509 certificates where possible.
+- Be using existing standards and and technologies where possible.
+
+### Terminology
+
+Throughout this document the following terminology is used:
+
+- Entity: An actor in the system (person or organization)
+- Agent: A digital representation of an entity that acts on its behalf, also often called an identity wallet
+- Vendor: A party that offers Agents to its customers (care organizations and healthcare professionals)
+- Identity: A set of claims about an entity (person or organization) which is relevant in a specific context
+- Claim: A statement about an entity (e.g. name, role, affiliation)
+- Verifiable Claim: A claim that can be cryptographically verified
+- Client: An entity that requests access to a resource on behalf of another entity
+- Issuer: An entity that issues claims about another entity
+- Authoritative source: An organization that is usually the authentic source of a claim and trusted to issue claims about a specific entity
+- Verifier: An entity that verifies the identity of another entity based on its claims
+- Holder: An entity that holds claims about itself or is authorized to present claims about another entity
+- Authentication: The process of verifying the identity of an entity
+- Authorization: The process of granting access to resources based on the identity of an entity
+- Use-case: A specified cross organization data exchange in which authentication and authorization requirements are defined
+
+### Proposed Authentication Approach
+
+This guide defines an authentication solution based on the _OAuth 2.0_ standard. It enables clients to obtain _Access Tokens_ using the extension defined in [RFC 7523](https://www.rfc-editor.org/rfc/rfc7523).
+
+In this approach, authentication of the participating entities is performed using a [JWT Authorization Grant](https://www.rfc-editor.org/rfc/rfc7523#section-2.1), while the client itself is authenticated through a [Client Authentication Assertion](https://www.rfc-editor.org/rfc/rfc7523#section-2.2).
+
+The _Authorization Grant_ carries verifiable identity claims for the involved parties (e.g., a healthcare provider and care giver), and the _Client Authentication Assertion_ conveys the client’s own identity.
+
+Identity claims are long-lived and issued in advance by multiple authoritative sources (such as registries responsible for maintaining identity information). These claims are represented in a cryptographically verifiable format compliant with the [Verifiable Credentials Data Model 1.1](https://www.w3.org/TR/2022/REC-vc-data-model-20220303/).
+
+Each use case may define its own required set of claims and the corresponding trust framework governing how entities interact. _Authorization Services_ can validate the presented claims without contacting the authoritative sources at the time of authentication.
```

Comment:
I'd call them "Authentication Services"

Reply:
Authentication Services is not the same as an authoritative source right?

Also, revocation lists can be cached and are anonymous.